script/client

```diff
@@ -205,7 +228,22 @@ elsif ($options{'json-data'}) {
     $res = $client->$method($url, {'Content-Type' => 'application/json'} => $options{'json-data'})->res;
 }
 else {
-    $res = $client->$method($url)->res;
+    # Either the user wants to call a command or wants to interact with
+    # the rest api directly.
+    if ($options{archive}) {
+        $options{path}    = $path;
+        $options{url}     = $url;
+        $options{params}  = \%params;
+        $options{params2} = @ARGV;
+        eval { $res = $client->archive->run(\%options) };
+        die "ERROR: $@ \n", $@ if $@;
+        exit(0);
+    }
+    else {
+        $res = $client->$method($url)->res;
+        #exit(0);
+    }
+
```

Comment:
Is it a conscious choice to add a blank line after and before each method end? I personally prefer to save visual space and not having blank lines for separation is one of the first things I avoid :)

Reply:
It's concious, perhaps not 100% consistent xD